### PR TITLE
Create cut term for go string function

### PR DESCRIPTION
**Pull request by 4hbab**
### Description

- Added strings.Cut() term for go
- Closes Issue #2562 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/string-functions/terms/cut/cut.md
Comment (SSwiniarski) [inline content/go/concepts/string-functions/terms/cut/cut.md:51]: We'll have to remove this codebyte because it doesn't work with the Codecademy codebytes. The codebyte compiler doesn't recognize the `.Cut` function.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@4hbab I did a review and had one change.

**Comment by 4hbab** [comment]
I ran the example code from the docs and that doesn't work either on the codeacademy codebytes. How can i fix the problem?

**Comment by SSwiniarski** [comment]
@4hbab at the moment all we can do is remove the codebyte.

**Review by SSwiniarski — APPROVED** [review]
@4hbab Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@4hbab @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@4hbab I didn't notice this before I merged, but the path of this entry is wrong. It should have been under strings, not string-functions. I submitted a PR to rectify this.

**Pull request merged**
